cachito/workers/pkg_managers/npm.py

```diff
@@ -192,6 +193,10 @@ def convert_to_nexus_hosted(dep_name, dep_info):
         # When the dependency is uploaded to the Nexus hosted repository, it will be in the format
         # of `<version>-gitcommit-<commit hash>`
         version_suffix = f"-external-gitcommit-{commit_hash}"
+        # Dangerous scripts might be required to be executed by `npm pack` since this is a Git
+        # dependency. If those scripts are present, Cachito will fail the request since it will not
+        # execute those scripts when packing the dependency.
+        verify_scripts = True
```

Comment:
Should this be a property of each dependency? It may be surprising if adding a git dependency, causes an error to an unrelated dep.

Reply:
I don't understand what you mean. This function is scoped to a single dependency.

Reply:
I misread the code.